nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -642,6 +642,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `services.xserver.desktopManager.xfce` now includes Xfce's screen locker, `xfce4-screensaver`.
 
+- The `radarr` service has a new option `package` to override the default package of Radarr.
```

Comment:
I don't think this is noteworthy, given that I'm clueless as to what you would override it with. If you want to use radarr from another channel you might as well go with an overlay.

Reply:
The option to override the used package seems to be more or less widely used. I would personally just use the package from another overlay (like unstable or master) without the need to override the options.

Reply:
The `package` attribute is pretty common in modules? I'm not inventing new anything here...

Many people (myself included) think overlays are to be avoided as much as possible for various reasons, see e.g.
https://discourse.nixos.org/t/what-are-overlays/14680/9
https://zimbatm.com/notes/1000-instances-of-nixpkgs

Of course I don't mind removing it from the release notes if you want :slightly_smiling_face:

Reply:
I was just using overlays as an example of how mundane this option is, so that in fact it is not release notes worthy.

Reply:
removed docs in https://github.com/NixOS/nixpkgs/pull/168141/commits/07f2a0ade4878d10929bb6076589b66382d349e4